sbws/core/scanner.py

```diff
@@ -169,10 +170,11 @@ def measure_relay(args, conf, destinations, cb, rl, relay):
         cb.controller, conf.getfloat('general', 'http_timeout'))
     # Pick a destionation
     dest = destinations.next()
+    # If there is no any destination at this point, it can not continue.
     if not dest:
-        log.warning('Unable to get destination to measure %s %s',
-                    relay.nickname, relay.fingerprint[0:8])
-        return None
+        log.critical("There are not any functional destinations.")
```

Comment:
Should we tell operators what to do to avoid this error?

For example:
"If you add multiple destinations to your config, sbws will use all the working destinations."

Reply:
Should we tell operators that multiple destinations are more reliable in the config or the documentation?

Reply:
Added comment to the config and doc, since dunno where operators look first.